server/routes/index.js

```diff
@@ -2,8 +2,8 @@ const express = require('express');
 
 const router = express.Router();
 
-router.get('/hi', (req, res) => {
-  res.send('hi');
-});
+const { signUp } = require('../controller');
+
+router.post('/signup', signUp);
```

Comment:
please see [iman PR comments](https://github.com/GSG-G9/beauty-salon/pull/53) to know how to organize the routes folder.